jerry-core/ecma/builtin-objects/ecma-builtin-json.cpp

```diff
@@ -126,9 +105,12 @@ ecma_builtin_json_check_id (lit_utf8_byte_t *string_p, /**< start position */
   {
     string_p++;
     id_p++;
-    if (*id_p == '\0')
+    if (*id_p == LIT_CHAR_NULL)
     {
-      return !isalpha (*string_p) && !isdigit (*string_p) && *string_p != '$' && *string_p != '_';
+      return (!lit_char_is_ascii_alpha (*string_p)
```

Comment:
@zherczeg, I'm trying to figure out what part / paragraph of ECMA-262 corresponds to the following conditions block:
```
(!lit_char_is_ascii_alpha (*string_p)
 && !lit_char_is_decimal_digit (*string_p)
 && *string_p != LIT_CHAR_DOLLAR_SIGN
 && *string_p != LIT_CHAR_UNDERSCORE)
```

Could you, please, clarify what is the paragraph / part?